framework/src/kernels/VectorFunctionReaction.C

```diff
@@ -9,7 +9,7 @@
 
 #include "VectorFunctionReaction.h"
 
-registerMooseObject("ElectromagneticsApp", VectorFunctionReaction);
+registerMooseObject("MooseApp", VectorFunctionReaction);
```

Comment:
for future reference, all these XXFunctionYY objects are easily outmatched by XXFunctorYY objects that can support functions, variables, functor material properties and even postprocessors. Scalar variable support soon

Reply:
no need to change anything here

Reply:
This is a good point. Moving some of the EM related objects to a functor configuration is still something on my roadmap from when we first merged the module in. I'll get to it at some point!

Reply:
Outmatched in flexibility, not in performance